compiler/options.nim

```diff
@@ -161,6 +161,9 @@ type
     version*: int
   Suggestions* = seq[Suggest]
 
+  NimprettyOpt* = object
```

Comment:
This option variable is already in options.nim. So would `Nimpretty` (without that suffix) serve as a better name?

Reply:
> This option variable is already in options.nim

sorry, where?

Reply:
I meant that you are declaring `NimprettyOpt` in `compiler/options.nim`. Isn't that "Opt" redundant then? And the same comment where you later create a variable of this object type.